client/my-sites/theme/main.jsx

```diff
@@ -573,6 +578,7 @@ class ThemeSheet extends React.Component {
 				href={ getUrl ? getUrl( this.props.id ) : null }
 				onClick={ this.onButtonClick }
 				primary={ isActive }
+				target={ this.props.isActive ? '_blank' : null }
```

Comment:
Minor, but `isActive` is already defined from `this.props` on line 573, so you can simply use `isActive` here rather than `this.props.isActive`.

Reply:
Done!